src/components/Colors.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+export const colors: { [key: string]: string } = {
+    'Unused memory': 'white',
+    'Sample 1': '#F44336',
+    'Sample 2': '#3F51B5',
+    'Sample 3': '#FFC107',
+    'Sample 4': '#9C27B0',
+    'Sample 5': '#4CAF50',
+    'Sample 6': '#673AB7',
+    'Sample 7': '#FF9800',
+    'Sample 8': '#E91E63',
+    'Sample 9': '#CDDC39',
+};
```

Comment:
Color samples used for both the memory visualization and the (currently unused) color legend